src/components/KTableView/ColumnVisibilityMenu.vue

```diff
@@ -114,28 +113,14 @@ import KInput from '@/components/KInput/KInput.vue'
 import KTooltip from '@/components/KTooltip/KTooltip.vue'
 import KLabel from '@/components/KLabel/KLabel.vue'
 
-const emit = defineEmits<{
-  (e: 'update', columnVisibility: Record<string, boolean>): void
-}>()
-
-const props = defineProps({
-  columns: {
-    type: Array as PropType<TableDataHeader[]>,
-    required: true,
-  },
-  tableId: {
-    type: String,
-    required: true,
-  },
-  visibilityPreferences: {
-    type: Object as PropType<Record<string, boolean>>,
-    default: () => ({}),
-  },
-  disabled: {
-    type: Boolean,
-    default: false,
-  },
-})
+const {
+  columns,
+  tableId,
+  visibilityPreferences,
```

Comment:
it looks like we lost the default here which could cause an error to be thrown when accessing properties below that aren't optionally chained?